### PR TITLE
feat: Landing page/ Home page

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Added landing page with navbar, different products, footer.
Also added images and icons used.

link: https://eloquent-jang-881e69.netlify.app/
You can preview below
image
image
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+.sidebar {
+  gap: 1rem;
+  width: 17%;
+  margin-left: 1rem;
+  padding: 1rem;
+  position: fixed;
+  background: var(--white);
+  height: 78%;
+}
```
Comment (JantuDeb) [inline css/style.css:221]: Do use media query for mobile

File: index.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                <div class="rel search-container">
+                    <input class="input-search" type="text" placeholder="Search Here" name="search" />
+                    <img class="search" src="./assets/icons-svg/search.svg" alt="search-icon" />
+                </div>
+            </li>
+            <ul class="navbar-list">
+                <li class="navbar-list-item"><button class="btn">Login</button></li>
+                <li class="navbar-list-item badge-container cursor">
+                    <img class="svg-icon" src="./assets/icons-svg/heart-white.svg" alt="heart" /><span
+                        class="badge badge-red">5</span>
```
Comment (JantuDeb) [inline index.html:33]: use code formatter before commit

**Review by JantuDeb — APPROVED** [review]
Suggested some changes

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+.img-banner {
+  width: 60%;
+  margin: 0 auto;
+  height: 28rem;
+  margin-top: 3rem;
+}
+
+.banner-content {
+  background: #283891;
+  color: white;
```
Comment (harshitadaswani) [inline css/style.css:96]: use variable

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+}
+
+.banner-badge {
+  position: absolute;
+  top: 0;
+  left: 0;
+  font-size: 3rem;
+  background: var(--red);
+  font-weight: bold;
+  color: white;
```
Comment (harshitadaswani) [inline css/style.css:108]: same

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+  padding: 4px;
+}
+
+.banner-text-container {
+  position: absolute;
+  background: #ffeca2;
```
Comment (harshitadaswani) [inline css/style.css:114]: same

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,302 @@
...
+  padding: 1rem;
+  background: var(--white);
+}
+
+.product-name {
+  color: #141111;
+  padding-left: 1rem;
+  font-weight: bold;
+  margin: 0;
+  margin-top: 0.5rem;
```
Comment (harshitadaswani) [inline css/style.css:246]: combine both into one

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,302 @@
...
+  font-weight: bold;
+}
+
+.product-action {
+  display: flex;
+  justify-content: space-between;
+}
+
+.product-buy-button {
+  background: #ed1f27;
```
Comment (harshitadaswani) [inline css/style.css:262]: use variable

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,302 @@
...
+  border: none;
+  padding: 1rem;
+  margin: 1rem;
+  font-size: 1.2rem;
+  border-radius: 5px;
+  color: white;
+}
+
+.product-cart-button {
+  color: #ed1f27;
```
Comment (harshitadaswani) [inline css/style.css:272]: use variable for colors

**Review by harshitadaswani — APPROVED** [review]
Refactor the mentioned suggestions and you are good to go

**Pull request merged**